src/sync-methods.ts

```
@@ -81,3 +81,11 @@ export const biChain =
   ) =>
   (result: Result<T, E>): Result<TS | ES, TF | EF> =>
     result.biChain(okFn, errFn);
+
+export const asTuple: {
+  <T, E>(
+    result: Result<T, E>,
+  ):
+    | readonly [ok: true, value: T, error: undefined]
+    | readonly [ok: false, value: undefined, error: E];
```

Biome

[error] Use a function type instead of a call signature.

Types containing only a call signature can be shortened to a function type.
Safe fix: Use a function type instead of an object type with a call signature.


(lint/style/useShorthandFunctionType)